src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      calculator.result += num;
+
+      return calculator;
+    },
```

Comment:
```suggestion
    add(num) {
      this.result += num;
    },
```
fix this comment. same for other operation methods
https://github.com/mate-academy/js_advanced_calculator/pull/1831/files#r1346972720